### PR TITLE
<ListItemSection/> - fix title and subheader colors

### PR DESCRIPTION
**Pull request by gendeld**
This fixes the issue reported here:
https://github.com/wix/wix-style-react/issues/5786

### 🏗 Relevant Components

`<ListItemSection/>`

### 😯 Current Behaviour

The title and subheader colors seem different than the zeplin spec:
**Spec:**
spec

**Production:**
production

Title color in spec: `#7a92a5` (D40).
Title color in production: `#162d3d`

Subheader color in spec: `#32536a` (D20).
Subheader color in production: `#162d3d`

### 👀 Severity
Low

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5841/

File: src/ListItemSection/ListItemSection.st.css [hunk 1]
```diff
@@ -58,9 +67,11 @@
...
   overflow: hidden;
 }
+
 .suffix:ellipsis span {
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
 /* TODO - remove once https://github.com/wix/wix-style-react/issues/4612 is resolved */
```
[the remark this answers is not in the document]
Comment (gendeld) [reply src/ListItemSection/ListItemSection.st.css:77]: Will address in separate PR :)

File: src/ListItemSection/ListItemSection.st.css [hunk 2]
```diff
@@ -27,11 +32,15 @@
 }
 
 .title {
+  -st-extends: Text;
   -st-states: subheader;
```
[the remark this answers is not in the document]
Comment (gendeld) [reply src/ListItemSection/ListItemSection.st.css:36]: Without it I get `unknown pseudo-state "skin"` on `.title:skin(standard)`

[the remark this answers is not in the document]
Comment (TalZa1810) [reply src/ListItemSection/ListItemSection.st.css:36]: I meant removing the `subheader` state, not the Text extend.

**Review by EgoziE — APPROVED** [review]

**Pull request merged**